assets/js/blocks/store-notices/edit.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import { Notice } from '@wordpress/components';
+
+const Edit = (): JSX.Element => {
+	const blockProps = useBlockProps( {
+		className: 'wc-block-store-notices',
+	} );
+
+	return (
+		<div { ...blockProps }>
+			<Notice status="warning" isDismissible={ false }>
```

Comment:
Another mistake in my suggestion again, I think we should use the `info` here instead of `warning`.
```suggestion
			<Notice status="info" isDismissible={ false }>
```

Reply:
Look like there is an issue with the style of information notice which make the background white. I think we should fix the issue upstream in Gutenberg and continue using the `info` notice. Otherwise, we can use `success` notice which doesn't have issue, it's up to you @albarin to make the decision, either is good to me.

Reply:
I'll use `info`, makes more sense for me as well.